src/elements/Input/input.less

```diff
@@ -81,4 +81,12 @@
 		top: 50%;
 		margin-top: -3px;
 	}
+
+	&.bare {
+		display: none;
```

Comment:
The `bare` class will cause the entire select wrap to be display none, is that the intent?